src/main/java/org/opensearch/plugin/insights/rules/model/SearchQueryRecord.java

```diff
@@ -41,6 +37,12 @@ public class SearchQueryRecord implements ToXContentObject, Writeable {
     private final long timestamp;
     private final Map<MetricType, Measurement> measurements;
     private final Map<Attribute, Object> attributes;
+    /**
+     * UUID
+     */
+    private final String id;
+    public static final String ID = "id";
```

Comment:
We already have ID in this file : https://github.com/opensearch-project/query-insights/pull/189/files#diff-4f0f5f144ba2dc04564479bc043edac327cfdbb4d2532aecb52092c8882b308bR102

Not sure why this is needed.

Reply:
based on our discussion will rename the older ID to query_group_id